configure.in:1134: error: required file 'src/libscg.a' not found
configure.in:1134: error: required file 'src/libschily.a' not found

--- e-uae-0.8.29-WIP4/configure.in
+++ e-uae-0.8.29-WIP4/configure.in
@@ -1162,13 +1162,7 @@
         SCSIOBJS="scsiemul.o blkdev.o blkdev-libscg.o"
         UAE_DEFINES="$UAE_DEFINES -DSCSIEMU"
         NEED_THREAD_SUPPORT=yes
-        if [[ "$srcdir" != "." ]]; then
-          AC_CONFIG_LINKS(src/libscg.a:src/libscg.a)
-          AC_CONFIG_LINKS(src/libschily.a:src/libschily.a)
-	  UAE_LIBS="$UAE_LIBS $LIBSCG_LIBS"
-        else
-          UAE_LIBS="$UAE_LIBS -L. $SCGLIBS"
-        fi
+	UAE_LIBS="$UAE_LIBS $LIBSCG_LIBS"
       ],
       [
         AC_MSG_RESULT(no)
